packages/schema/package.json

```diff
@@ -25,24 +25,22 @@
 	"exports": {
 		".": "./dist/index.js",
 		"./package.json": "./package.json",
-		"./types/overview": {
-			"types": "./dist/types/overview.d.ts",
-			"default": "./dist/types/overview.js"
-		}
+		"./types/overview": "./dist/types/overview.js"
 	},
```

Comment:
why are we explicitly bundling `./types/overview.js` separately here? Should we not export this from the root too? 🤔

Reply:
Good catch! This is a legacy thing. Although unrelated, I opted to clean that up in this PR 👍